src/binary.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+import {broadcast, getBroadcastShape} from './broadcast.js';
+import {Tensor, sizeOfShape} from './tensor.js';
+
+/**
+ * Compute the element-wise binary operation of two input tensors.
+ * @param {Tensor} inputA
+ * @param {Tensor} inputB
+ * @param {Function} binaryFunc
+ * @return {Tensor}
+ */
+function binary(inputA, inputB, binaryFunc) {
+  const outputShape = getBroadcastShape(inputA.shape, inputB.shape);
+  const inputABroadcast = broadcast(inputA, outputShape);
```

Comment:
@huningxin Should we consider optimizing of memory usage here?

Reply:
Thanks for this consideration. Code readability and algorithm simplicity are focus of the baseline implementation. If we see issues later, we can optimize.